### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by VedanthB**
- added lists
- added lists with avatar, for notifications 

https://user-images.githubusercontent.com/75097551/152406981-4a79e218-db83-43ed-9c52-767bc51e200c.mov

### CODE REVIEW

**Review by Ananyamadhu08 — APPROVED** [review]
looks good 🚀

File: index.html [hunk 1]
```diff
@@ -323,6 +323,253 @@ <h2 class="h2">opacity</h2>
...
+
+          <label for="email" class="input-label text-grey-700 bg-grey-100"
+            >Email *
+          </label>
+
+          <small class="error text-red-500 text-sm">Error Message</small>
+        </div>
+      </form>
+    </div> -->
+    <div class="test-2">
```
Comment (dev-enforced) [inline index.html:517]: Suggestions:
Do add a comment here in order to track out where the Lists related component details are!!

File: styles.css [hunk 2]
```diff
@@ -8,3 +8,8 @@
   margin: 10px;
   padding: 10px;
 }
+
+.test-2 {
+  margin: 50px;
```
Comment (dev-enforced) [inline styles.css:13]: You can use rems measurements here in the margin and padding too!!

File: index.html [hunk 3]
```diff
@@ -323,6 +323,253 @@ <h2 class="h2">opacity</h2>
...
+      </div>
+    </div>
+    <div class="test-2 w-80">
+      <div class="flex flex-col shadow-md rounded-md">
+        <div class="badge-container img-container">
+          <img
+            class="rounded-md img-responsive"
+            src="https://picsum.photos/350/250"
+            alt=""
+            srcset="https://picsum.photos/350/250"
```
Comment (dev-enforced) [inline index.html:400]: Suggestions:
src and srcset are relating to the same image link so you can keep either one of them no need to keep both
Put a value in the alt attribute

**Review by dev-enforced — CHANGES_REQUESTED** [review]
Suggestions have been mentioned below !!
You can check it out.

File: index.html (hunk 3 above)
Comment (VedanthB) [reply index.html:400]: okay will make the changes

File: styles.css (hunk 2 above)
Comment (VedanthB) [reply styles.css:13]: these are just test classes i will delete them

File: index.html (hunk 1 above)
Comment (VedanthB) [reply index.html:517]: yes, will do that

**Pull request merged**